src/script/event/EventRepository.ts

```diff
@@ -186,6 +180,8 @@ export class EventRepository {
     await this.handleTimeDrift();
     const connect = () => {
       this.notificationHandlingState(NOTIFICATION_HANDLING_STATE.STREAM);
+      // We make sure there is only be a single active connection to the WebSocket.
+      this.disconnectWebSocket?.();
```

Comment:
To be on the safe side we want to make absolutely sure we are never connected to 2 websockets at the same time